aws-kendra-faq/src/test/java/software/amazon/kendra/faq/UpdateHandlerTest.java

```diff
@@ -73,6 +73,7 @@ public void handleRequest_SimpleSuccess() {
         String name = "name";
         String s3Key = "s3Key";
         String s3Bucket = "s3Bucket";
+        String fileFormat = "CSV";
```

Comment:
Same as above do we have case where this is not provided?

Reply:
Added new test.